starboard/android/shared/media_decoder.cc

```diff
@@ -534,6 +545,7 @@ void MediaDecoder::HandleError(const char* action_name, jint status) {
     SB_LOG(INFO) << "|" << action_name << "| failed with status: "
                  << GetNameForMediaCodecStatus(status)
                  << ", will try again after a delay.";
+    SbThreadSleep(kErrorRetryDelay);
```

Comment:
The log message says 'will try again after a delay', but there is no actual delay. When testing this, this would make the thread spin, unnecessarily loading a CPU core.

So minimize the impact on fast startups, I could make this progressive, starting at no delay.

Reply:
For now, changed this to a yield instead of a sleep. May followup with some sleeping strategy.